packages/stark-build/config/webpack-partial.common.js

```diff
@@ -24,6 +25,14 @@ const BundleAnalyzerPlugin = require("webpack-bundle-analyzer").BundleAnalyzerPl
 // const InlineManifestWebpackPlugin = require("inline-manifest-webpack-plugin");
 // const ScriptExtHtmlWebpackPlugin = require("script-ext-html-webpack-plugin");
 
+const tslintConfig = buildUtils.get(buildUtils.ANGULAR_APP_CONFIG.config, "architect.lint.options.tslintConfig", "tslint.json");
+const tslintConfigPath = helpers.root(tslintConfig).replace(/\\/g, "/");
+const tslintLoaderFilePath = `${os.tmpdir()}/national-bank-belgium_stark-build_tslint-${new Date().getTime()}.json`;
+
+const tslintDefault = fs.readFileSync(helpers.rootStark("config/tslint-disabled-typecheck-rules.json"), "utf8");
```

Comment:
Can you rename this to something more descriptive... for example `tslintConfigNoTypeChecks`